carma-messenger-core/carma-messenger/launch/carma-messenger.launch

```diff
@@ -31,7 +31,8 @@
   <env name="JAVA_OPTS" value="$(arg JVM_LOG_OPTS)"/>
   <!-- Generate log name configuration file -->
   <param name="log_name" type="str" command="$(find carma-messenger)/scripts/generate_log_name.sh"/>
-
+    <remap from="outbound_binary_msg" to="comms/outbound_binary_msg"/>
```

Comment:
I think you still need this change correct and not related to https://github.com/usdot-fhwa-stol/carma-messenger/pull/45?

Reply:
Please remove the unnecessary changes on the other files if you need this one pushed.

Reply:
https://github.com/usdot-fhwa-stol/carma-messenger/pull/49/commits/5272780616a8aad5b79d165058571456dfd8a19b

Reply:
We need to move the remap to the top because both cpp_message node and java message node use this topics remapping